pkg/cli/zip.go

```diff
@@ -64,7 +64,7 @@ var debugZipTablesPerCluster = []string{
 	"system.jobs",       // get the raw, restorable jobs records too.
 	"system.descriptor", // descriptors also contain job-like mutation state.
 	"system.namespace",
-	"system.namespace_deprecated", // TODO(sqlexec): consider removing in 20.2 or later.
+	"system.namespace_with_parent_schema", // TODO(sqlexec): consider removing in 20.2 or later.
```

Comment:
is it worth keeping `system.namespace_deprecated` alongside this, so that we get a full picture for those few customers that happen to have both? (Unless you have some confidence that your PR prevents this from even being possible)